src/plugin-hooks/useRowSelect.js

```diff
@@ -153,6 +153,7 @@ function reducer(state, action, previousState, instance) {
 
     const handleRowById = id => {
       const row = rowsById[id]
+      if (row === undefined) return
```

Comment:
from looking through some other files, i think it might be a repo convention to include the `{` `}` like this:

```suggestion
      if (row === undefined) {
        return
      }
```